### ISSUE TITLE
lastRow: 'justify' ignores fixedHeight: false

### ISSUE THREAD

**Issue by rulatir** [miromannino/Justified-Gallery#98]
When the combination lastRow: 'justify', fixedHeight: false is used, the fixedHeigh: false setting is ignored for the last row.

Actually the <img> elements have correct heights calculated from the aspect ratio (I mean CSS height in the style attribute), but their surrounding <a> elements have the CSS height forcibly set to the value of rowHeight.

Is there some hidden knob that will just make it work or do I have to hack?

**Comment by miromannino** [comment]
I cannot replicate the error, it works well with `{ lastRow: 'justify', fixedHeight: false}`. The last row is justified and the images are very bigger.

The only thing that could influence this is a setting that impose a maximum size: `maxRowHeight`.
By default, it impose the row to not exceed the rowHeight by 150%. This means that if the maxRow is 100px it impose to not exeed 150px, having the effect you mentioned.

Try to set `maxRowHeight : -1`. It works well for you?

That indeed is an usability problem! If you asked me if it is a bug, for sure that's could be considered as an usability/documentation problem, and I thank you contacted me.

I will disable this default behaviour in 3.6, setting maxRowHeight: -1 by default.

Best regards,

Miro

_If this project is useful for you and since I'm working on it for free during my midnights, please consider to contribute in any way. Take few minutes to share this project or to add me on LinkedIn (Miro Mannino) to endorse me some skills or to leave me a positive comment (or just to connect each other in case of future projects together). I don't want donations, I just would to be recognised for my works. Thank you_

**Comment by rulatir** [comment]
I had set maxRowHeight to 1, but the actual problem was that FooGallery for Wordpress bundles jG v3.2 which didn't have maxRowHeight yet. I figured it out in the meantime, replaced the bundled version with the current version, and it JustWorks™.

Your attention to usability and documentation clarity is commendable. I am not exactly well-networked (I don't do LinkedIn) but I'll think of some way to endorse you. You'd get 5 stars from me if this were a Wordpress plugin.

Btw. I have implemented a basic version of this this algorithm on several occasions so I can definitely appreciate.
